engine/security/rootless.md

```diff
@@ -131,9 +136,9 @@ $ curl -fsSL https://get.docker.com/rootless | sh
 ```
 
 Make sure to run the script as a non-root user.
-To install Rootless Docker as the root user, see [Manual installation](#manual-installation) steps.
+To install Rootless Docker as the root user, see the [Manual installation](#manual-installation) steps.
 
-The script will show the environment variables that are needed to be set:
+The shows environment variables that are required:
```

Comment:
The script shows ...

Reply:
Aah thanks for spotting this @AkihiroSuda :)

Reply:
Fixed now